config.json

```diff
@@ -35,6 +35,17 @@
   },
   "exercises": {
     "concept": [
+      {
+        "slug": "annalyns-infiltration",
+        "name": "Annalyn's Infiltration",
+        "uuid": "897fa2ea-c29c-4d4e-9469-ff661a9b838a",
```

Comment:
Unfortunately concept exercises [just got disabled](https://github.com/exercism/rust/commit/af8e3cfa23a179644d6321df43a92c7915676885) for Rust. The syllabus will need a complete rework, so it might take some time to enable this concept exercise. (CC @ErikSchierboom)

Reply:
We will show in-progress exercise for track maintainers though.